.github/workflows/qa.yml

```diff
@@ -16,12 +16,14 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v3
       with:
-        python-version: "3.10"
+        python-version: "3.10.6"
```

Comment:
Why this change?

Reply:
CPython 3.10.7 breaks mypy. The fix has been already added to the changes for the next version, but since it hasn't been released yet, this is required for now.

Reply:
I added the 0.980 release branch for mypy in the requirements so this change is not longer needed. Removed